### PR TITLE
Add JSON_SCHEMA_VALID

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Document `JSON_SCHEMA_VALID()`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

https://github.com/pingcap/tidb/pull/52780

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: functions-and-operators/json-functions.md [hunk 1]
```diff
@@ -70,14 +70,19 @@ TiDB supports most of the [JSON functions](https://dev.mysql.com/doc/refman/8.0/
 | [JSON_ARRAYAGG(key)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-arrayagg) | Provides an aggregation of keys. |
 | [JSON_OBJECTAGG(key, value)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-objectagg) | Provides an aggregation of values for a given key. |
 
+## Validation functions
+
+| Function Name                     | Description |
+| --------------------------------- | ----------- |
+| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validate a JSON document against a schema. |
```
Comment (hfxsd) [inline functions-and-operators/json-functions.md:77]:
```suggestion
| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validate a JSON document against a schema to improve data integrity and consistency. |
```

Comment (dveeden) [reply functions-and-operators/json-functions.md]: Updated in f198d36b8833f4dd60f7a4b8ad26083a84f5fb6e 

I don't think "to improve" is correct here as the function only checks the data and doesn't change (improve) the data.

PTAL @hfxsd

**Review by lilin90 — APPROVED** [review]
/lgtm

File: functions-and-operators/json-functions.md [hunk 2]
```diff
@@ -70,14 +70,19 @@ TiDB supports most of the [JSON functions](https://dev.mysql.com/doc/refman/8.0/
 | [JSON_ARRAYAGG(key)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-arrayagg) | Provides an aggregation of keys. |
 | [JSON_OBJECTAGG(key, value)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-objectagg) | Provides an aggregation of values for a given key. |
 
+## Validation functions
+
+| Function Name                     | Description |
+| --------------------------------- | ----------- |
+| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validate a JSON document against a schema to ensure data integrity and consistency. |
```
Comment (hfxsd) [inline functions-and-operators/json-functions.md:77]:
```suggestion
| [JSON_SCHEMA_VALID(schema,json_doc)](https://dev.mysql.com/doc/refman/8.0/en/json-validation-functions.html#function_json-schema-valid) | Validates a JSON document against a schema to ensure data integrity and consistency. |
```

**Pull request merged**